pkg/utils/yamlhelper.go

```diff
@@ -28,21 +28,23 @@ func getYamlOfObject(object interface{}) (string, error) {
 	}
 }
 
-func generateYamlDiff(yaml1 string, yaml2 string) string {
+func generateYamlDiff(yaml1 string, yaml2 string, context int) string {
 	diff, _ := difflib.GetUnifiedDiffString(difflib.UnifiedDiff{
 		A:        difflib.SplitLines(yaml1),
 		B:        difflib.SplitLines(yaml2),
 		FromFile: "Old",
 		FromDate: "",
 		ToFile:   "New",
 		ToDate:   "",
-		Context:  1,
+		Context:  context,
 	})
 	return diff
 }
 
 // func to generate yaml diff of objects used for hashing
-func GetYamlDiffForObjects(objectOld interface{}, objectNew interface{}) (string, error) {
+// context : number of context lines to use for generating diff
+// for more reference on context : https://github.com/pmezard/go-difflib/blob/5d4384ee4fb2527b0a1256a821ebfc92f91efefc/difflib/difflib.go#L559
```

Comment:
That link is not very useful because reading the details of diff is not really helping to explain what context is. I would either leave it out entirely or replace with https://www.gnu.org/software/diffutils/manual/html_node/Unified-Format.html#Unified-Format because it explains what context lines are.

Reply:
Thanks, I will update.